### ISSUE TITLE
Initiative: Bug fixes

### ISSUE THREAD

**Issue by tobiasahlin** [github/mona-sans#31]
We have yet to commit to an exact scope for v1.1, but we want to make sure to squash the most critical bugs. 

These are the ones that have been reported so far:
- https://github.com/github/mona-sans/issues/27
- https://github.com/github/mona-sans/issues/25
- https://github.com/github/mona-sans/issues/21
- https://github.com/github/mona-sans/issues/20
- https://github.com/github/mona-sans/issues/16
- https://github.com/github/mona-sans/issues/7
- https://github.com/github/mona-sans/issues/6

**Comment by ottta** [comment]
Is it open for collaborator?

**Comment by tobiasahlin** [comment]
@ottta Yes! We have a round of bundled updates just about to land in a PR, but after that it's a great opportunity to branch from that state 😌 they'll go in through the #9 PR.
